### ISSUE TITLE
istioctl manifest generate gives duplicate resources

### ISSUE THREAD

**Issue by Stono** [istio/istio#21718]
**Bug description**
Trying to use `1.5.0-beta5` against my existing values file, and the output from `istioctl manfiest generate` gives me this resource 3 times:

```
apiVersion: apiextensions.k8s.io/v1beta1
kind: CustomResourceDefinition
metadata:
  annotations:
    "helm.sh/resource-policy": keep
  labels:
    app: mixer
    chart: istio
    heritage: Tiller
    istio: mixer-instance
    package: instance
    release: istio
  name: instances.config.istio.io
```

**Expected behavior**
To get the same output as helm template. 

**Steps to reproduce the bug**
can provide my values file on slack

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**

**How was Istio installed?**

**Environment where bug was observed (cloud vendor, OS, etc)**

**Comment by Stono** [comment]
John just linked me this https://github.com/istio/istio/pull/20775

**Comment by howardjohn** [comment]
This was fixed in the 1.5.0 release by https://github.com/istio/istio/pull/20775 I believe
